fred2/management.cpp

```diff
@@ -462,6 +462,12 @@ bool fred_init(std::unique_ptr<os::GraphicsOperations>&& graphicsOps)
 
 	Script_system.RunCondition(CHA_GAMEINIT);
 
+	// wookieejedi
```

Comment:
This should be moved prior to `gr_reset_clip();` because that's where the "init" functionality concludes and the "clean up" code starts.

For qtFRED, move it to immediately after `sexp::dynamic_sexp_init();`.

Reply:
Thanks, just changed it.